Makefile

```diff
@@ -99,17 +99,17 @@ test-unit-go:
 build: build-images xbuild-cli
 
 .PHONY: build-images
-build-images: build-apiserver
+build-images: build-apiserver build-scheduler
```

Comment:
Just a reminder that last time I tried building with `SKIP_DOCKER=true`, it was still trying to use Docker for some targets.

Reply:
What targets? SKIP_DOCKER only prevents non-docker commands (like go test) from running in a container. Commands that build images will still use docker.

Reply:
Sure, I think I was trying to run the Go tests outside a container (I will try again when properly reviewing this PR).

Reply:
Ultimately, what I would very much like are targets to build code locally, without a container.

Sure, targets to build images have to use Docker, but while working on the project locally, it would be very helpful to be able to do this.

Reply:
> Ultimately, what I would very much like are targets to build code locally, without a container.

That should already be possible with `SKIP_DOCKER=true`. If that's not working for some reason, let's fix that rather than give up on the option to do things within a container.

fwiw, I understand not wanting to always use containers if they are slowing you down, but there's definitely a benefit to being able to execute things such as tests and CLI builds in an environment that's identical to where those things will run in CI, so I don't want to lose that.

Reply:
I definitely wouldn't suggest stopping building in containers, but for example, right now it's impossible to build the API server locally without Docker.